### PR TITLE
small fix on creating the toolchainCluster with expected type label

### PR DESCRIPTION
**Pull request by mfrancisc**
create `toolchainCluster` resource in `await.Namespace` with `otherAwait.Type` .

In the scenario where `await.Namespace` is `toolchain-host-ns`, the type label for the `toolchainCluster` resource should be `otherAwait.Type` (so `member` in this case)

### CODE REVIEW

**Review by xcoulon — APPROVED** [review]

**Comment by mfrancisc** [comment]
/retest

infra issue

File: test/e2e/toolchaincluster_test.go [hunk 1]
```diff
@@ -36,7 +36,7 @@ func verifyToolchainCluster(t *testing.T, await *wait.Awaitility, otherAwait *wa
 		// given
 		name := "new-ready-" + string(await.Type)
```
Comment (MatousJobanek) [inline test/e2e/toolchaincluster_test.go:37]: this one as well right? 
```suggestion
		name := "new-ready-" + string(otherAwait.Type)
```

**Review by MatousJobanek — COMMENTED** [review]
Thanks for fixing it.
could you please fix it also in the following test? 
https://github.com/codeready-toolchain/toolchain-e2e/blob/5549f388e2a0bd72ab0c33fdbe1dad2a4f8b1504/test/e2e/toolchaincluster_test.go#L67-L71

File: test/e2e/toolchaincluster_test.go (hunk 1 above)
Comment (mfrancisc) [reply test/e2e/toolchaincluster_test.go:37]: ah right! missed that! thx!

**Pull request merged**